app/preprints/-components/submit/author-assertions/component.ts

```diff
@@ -24,6 +24,10 @@ interface AuthorAssertionsForm {
     hasDataLinks: string;
     whyNoData: string;
     dataLinks: string[];
+    hasPreregLinks: string;
+    whyNoPrereg: string;
+    preregLinks: string[];
+    preregLinkInfo: string[];
```

Comment:
Same thing here for using a PreregLinkInfoEnum here. Also, this isn't a string[] is it?